python/test/unit/language/test_core.py

```diff
@@ -465,7 +465,6 @@ def test_floordiv(dtype_x, dtype_y, num_ctas, device):
     _test_binary(dtype_x, dtype_y, expr, numpy_expr, device=device, num_ctas=num_ctas)
 
 
-@pytest.mark.interpreter
 @pytest.mark.parametrize("expr", ["tl.math.fdiv(x, y)", "tl.math.div_rn(x, y)"])
```

Comment:
Anything wrong with the interpreter?

Reply:
Was getting the error
```
E   NotImplementedError: 
E   fdiv not supported in interpreter mode: no known numpy implementation.
E   If you think that fdiv in fact does have a numpy implementation, please add it
E   to the mapping in python/triton/runtime/interpreter.py:_patch_lang_math.
```

But fixing was easy enough, so I've restored the interpreter test.

Reply:
Nice, thanks